src/connectors/nrk-radio.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-Connector.playerSelector = '.mini-player';
+Connector.playerSelector = '.app';
 
-Connector.artistTrackSelector = '.currently-playing .episode-title';
+Connector.artistTrackSelector = '.css-6752n5.e79qwfn0';
```

Comment:
This looks very brittle - like a new release of their site will cause things to break. Could we not use the `data-test="playingEpisodeDesc"` attribute to resolve this information?